### ISSUE TITLE
Sentence Similarity web demo is not working

### ISSUE THREAD

**Issue by ibrahimsharaf** [explosion/spaCy#1406]
Hi, I am not sure if it's the right repo to report this issue, but I was tinkering around with spaCy sentence similarity demo and it is not working on all my browsers, it keeps loading and giving me this:

> Oops, something went wrong. Please try again!

Example query: https://demos.explosion.ai/similarity/?text1=Spain%20&text2=The%20Philippines

**Comment by lgenerknol** [comment]
The cause is `GET https://api.explosion.ai/similarity/models` returns a 502.
Additionally, the root `GET https://api.explosion.ai/` returns the default nginx page.

**Comment by msamogh** [comment]
Any update on this? 

I'm unable to run the siamese network provided in the thinc repo's demo folder as well. So hoping for some solution to this to test out how well it works.

**Comment by honnibal** [comment]
Sorry for dragging my feet on this folks.

I was never really thrilled by the results the similarity models were giving, even though the performance on the development data was pretty good. I think the things the models learned really didn't generalise well.

I've now replaced the similarity demo with one that compares the similarity results from spaCy's `en_core_web_lg` and `en_core_web_sm` models. I'm hoping to revisit the supervised similarity stuff soon.

@msamogh I thought I got the siamese example up to date in thinc? When did you try?

**Comment by msamogh** [comment]
@honnibal I tried it a couple of times in the last month. 

There was a one-line change that I had to make in Thinc's source code to get the example running. When I tried recompiling, however, I would run into a circular dependency error involving the spaCy and Thinc versions. So I could never get it to work.

Regarding this:
> I was never really thrilled by the results the similarity models were giving, even though the performance on the development data was pretty good. I think the things the models learned really didn't generalise well.

Was it trained on the Quora QA dataset?
